docs/application/native/guides/app-management/common-appcontrol.md

```diff
@@ -2239,6 +2239,83 @@ vpn_appcontrol_result_cb(app_control_h request, app_control_h reply, app_control
     app_control_get_extra_data(reply, APP_CONTROL_DATA_TEXT, &result_txt);
     dlog_print(DLOG_INFO, LOG_TAG, "Result: %s", result_txt);
 }
+```
+## Privacy setting guide popup
+
+The Privacy setting guide popup application control is supported only in wearable applications since Tizen 5.0.
+
+### Shows privacy related common guide popup
+
+To show common privacy setting guide popup, use the `APP_CONTROL_OPERATION_PRIVACY_SETTING_GUIDE` operation. This operation helps you to guide users to change privacy privilege settings. You need to check permissions to see if the user actually granted requested permissions after using this application control.
```

Comment:
Suggestion:

Rephrase the last sentence to:
After using this application control, check permissions to verify whether the user is granted with the requested permissions.

Please verify that the technical meaning is not changed.

Reply:
Not the user who's getting a permission. It's the application. User can grant a permission to the application to perform specific function. How about "After using this application control, check permissions to verify whether the user granted the requested permissions." ? Need your opinion.

Reply:
Perfect!!
Sorry, I misinterpreted. Please go ahead and rephrase the sentence to "After using this application control, check permissions to verify whether the user granted the requested permissions."

Reply:
Done.